server/@types/ui/index.d.ts

```diff
@@ -74,11 +74,15 @@ export interface RadioItem {
   checked?: boolean
 }
 
-export interface CheckBoxItem {
-  text: string
-  value: string
-  checked?: boolean
-}
+export type CheckBoxItem =
+  | {
```

Comment:
Does this initial `|` do anything?

Reply:
Eslint puts in in automatically for `$REASONS`